src/components/CommandPalette.tsx

```diff
@@ -454,42 +454,29 @@ const CommandPalette: FC = () => {
               ...(!isTouch ? { maxHeight: 'calc(100vh - 8em)', overflow: 'auto' } : null),
             })}
           >
-            {shortcuts.length > 0 ? (
-              <>
-                {(() => {
-                  const helpCommand = commandById('help')
-                  // Check if current gesture matches any command's gesture once
-                  const hasMatchingCommand = shortcuts.some(cmd => gestureInProgress === cmd.gesture)
```

Comment:
`hasMatchingCommand` was accidentally removed.

Reply:
the commit didn't come through initially, sorry